telegram/inline/inlinequeryresultaudio.py

```diff
@@ -48,9 +48,9 @@ class InlineQueryResultAudio(InlineQueryResult):
         id (:obj:`str`): Unique identifier for this result, 1-64 bytes.
```

Comment:
In the base class we actually call `str(id)`, so `id (:obj:`str` | :obj:`int`)` would be okay for the argument. Just noticed while reviewing. Also for the other InlineQueryResult* classes

Reply:
Alright I'll change that in all InlineQueryResult* classes